air/src/stack/op_flags/mod.rs

```diff
@@ -0,0 +1,642 @@
+
+    #[inline(always)]
+    fn pad(&self) -> E {
+        self.degree7_op_flags[48]
+    }
+
+    #[inline(always)]
+    fn dup(&self) -> E {
+        self.degree7_op_flags[49]
+    }
+
+    #[inline(always)]
+    fn dup1(&self) -> E {
+        self.degree7_op_flags[50]
+    }
+
+    #[inline(always)]
+    fn dup2(&self) -> E {
+        self.degree7_op_flags[51]
+    }
+
+    #[inline(always)]
+    fn dup3(&self) -> E {
+        self.degree7_op_flags[52]
+    }
+
+    #[inline(always)]
+    fn dup4(&self) -> E {
+        self.degree7_op_flags[53]
+    }
+
+    #[inline(always)]
+    fn dup5(&self) -> E {
+        self.degree7_op_flags[54]
+    }
+
+    #[inline(always)]
+    fn dup6(&self) -> E {
+        self.degree7_op_flags[55]
+    }
+
+    #[inline(always)]
+    fn dup7(&self) -> E {
+        self.degree7_op_flags[56]
+    }
+
+    #[inline(always)]
+    fn dup9(&self) -> E {
+        self.degree7_op_flags[57]
+    }
+
+    #[inline(always)]
+    fn dup11(&self) -> E {
+        self.degree7_op_flags[58]
+    }
+
+    #[inline(always)]
+    fn dup13(&self) -> E {
+        self.degree7_op_flags[59]
+    }
+
+    #[inline(always)]
+    fn dup15(&self) -> E {
+        self.degree7_op_flags[60]
+    }
+
+    #[inline(always)]
+    fn read(&self) -> E {
+        self.degree7_op_flags[61]
+    }
+
+    #[inline(always)]
+    fn sdepth(&self) -> E {
+        self.degree7_op_flags[62]
+    }
+
+    // ------ Degree 6 u32 operations  --------------------------------------------------------
+
+    #[inline(always)]
+    fn u32add(&self) -> E {
+        self.degree6_op_flags[0]
+    }
+
+    #[inline(always)]
+    fn u32sub(&self) -> E {
+        self.degree6_op_flags[1]
+    }
+
+    #[inline(always)]
+    fn u32mul(&self) -> E {
+        self.degree6_op_flags[2]
+    }
+
+    #[inline(always)]
+    fn u32div(&self) -> E {
+        self.degree6_op_flags[3]
+    }
+
+    #[inline(always)]
+    fn u32split(&self) -> E {
+        self.degree6_op_flags[4]
+    }
+
+    #[inline(always)]
+    fn u32assert2(&self) -> E {
+        self.degree6_op_flags[5]
+    }
+
+    #[inline(always)]
+    fn u32add3(&self) -> E {
+        self.degree6_op_flags[6]
+    }
+
+    #[inline(always)]
+    fn u32madd(&self) -> E {
+        self.degree6_op_flags[7]
+    }
+
+    // ------ Degree 6 non u32 operations  ---------------------------------------------------
+
+    #[inline(always)]
+    fn rpperm(&self) -> E {
+        self.degree6_op_flags[8]
+    }
+
+    #[inline(always)]
+    fn mpverify(&self) -> E {
+        self.degree6_op_flags[9]
+    }
+
+    #[inline(always)]
+    fn span(&self) -> E {
+        self.degree6_op_flags[12]
+    }
+
+    #[inline(always)]
+    fn join(&self) -> E {
+        self.degree6_op_flags[13]
+    }
+
+    #[inline(always)]
+    fn split(&self) -> E {
+        self.degree6_op_flags[14]
+    }
+
+    #[inline(always)]
+    fn r#loop(&self) -> E {
```

Comment:
I'm guessing we need to do it like this because `loop` is a keyword. Maybe in this case would should do `loop_op` instead of `r#loop`?